roboflow/core/version.py

```diff
@@ -53,7 +57,7 @@ def __init__(self, version_dict, type, api_key, name, version, model_format, loc
             self.splits = version_dict["splits"]
             self.model_format = model_format
 
-            version_without_workspace = os.path.basename(version)
+            version_without_workspace = os.path.basename(str(version))
```

Comment:
this was failing if `version` was a number... which seems to be a thing that happens.

Reply:
nice catch!